simpl/rest.py

```diff
@@ -53,3 +57,142 @@ def wrapped(*args):
             return fxn(data, *args)
         return wrapped
     return wrap
+
+
+def write_body(data, request, response):
+    """Write output with format based on accept header.
+
+    This cycles through the global HANDLERs to match content type and then
+    calls that handler. Additional handlers can be added to support Additional
+    content types.
+    """
+    accept = request.get_header('Accept', ['application/json'])
+
+    for content_type in HANDLERS:
+        if content_type in accept:
+            return HANDLERS[content_type](data, request, response)
+
+    # Use default
+    return HANDLERS['default'](data, request, response)
+
+
+def write_json(data, request, response):
```

Comment:
We also had `write_yaml`and other HANDLERs and some used `request`, so this is more of a signature. This is a copy/paste, not new code, but I should document how this works and how to extend it. I'll do that...